src/styles/form-field.scss

```
@@ -0,0 +1,39 @@
+.form-field {
+  box-sizing: border-box;
+  display: block;
+
+  width: 100%;
+  height: 48px;
+  padding: 14px 16px;
+
+  color: $colorPlaceholderHover;
+
+  border: 1px solid transparent;
```

transparent is a color)
```suggestion
  border: 1px solid $c-transparent;
```